reference/manifest/createaddincommands.md

```diff
@@ -537,26 +459,25 @@ The following shows an example of how to use the **Resources** element. Each res
   
 
 |**Office host application**|**Tab values**|
 |:-----|:-----|
-|Excel  <br/> |**TabHome**         **TabInsert**         **TabPageLayoutExcel**         **TabFormulas**         **TabData**         **TabReview**         **TabView**         **TabDeveloper**         **TabAddIns**         **TabPrintPreview**         **TabBackgroundRemoval** <br/> |
-|Word  <br/> |**TabHome**         **TabInsert**         **TabWordDesign**         **TabPageLayoutWord**         **TabReferences**         **TabMailings**         **TabReviewWord**         **TabView**         **TabDeveloper**         **TabAddIns**         **TabBlogPost**         **TabBlogInsert**         **TabPrintPreview**         **TabOutlining**         **TabConflicts**         **TabBackgroundRemoval**         **TabBroadcastPresentation** <br/> |
-|PowerPoint  <br/> |**TabHome**         **TabInsert**         **TabDesign**         **TabTransitions**         **TabAnimations**         **TabSlideShow**         **TabReview**         **TabView**         **TabDeveloper**         **TabAddIns**         **TabPrintPreview**         **TabMerge**         **TabGrayscale**         **TabBlackAndWhite**         **TabBroadcastPresentation**         **TabSlideMaster**         **TabHandoutMaster**         **TabNotesMaster**         **TabBackgroundRemoval**         **TabSlideMasterHome**          <br/> |
+|Excel  <br/> |**TabHome**<br/>**TabInsert**<br/>**TabPageLayoutExcel**<br/>**TabFormulas**<br/>**TabData**<br/>**TabReview**<br/>**TabView**<br/>**TabDeveloper**<br/>**TabAddIns**<br/>**TabPrintPreview**<br/>**TabBackgroundRemoval** <br/> |
+|Word  <br/> |**TabHome**<br/>**TabInsert**<br/>**TabWordDesign**<br/>**TabPageLayoutWord**<br/>**TabReferences**<br/>**TabMailings**<br/>**TabReviewWord**<br/>**TabView**<br/>**TabDeveloper**<br/>**TabAddIns**<br/>**TabBlogPost**<br/>**TabBlogInsert**<br/> **TabPrintPreview**<br/>**TabOutlining**<br/>**TabConflicts**<br/>**TabBackgroundRemoval**<br/>**TabBroadcastPresentation** <br/> |
+|PowerPoint  <br/> |**TabHome**<br/>**TabInsert**<br/>**TabDesign**<br/>**TabTransitions**<br/>**TabAnimations**<br/>**TabSlideShow**<br/>**TabReview**<br/>**TabView**<br/>**TabDeveloper**<br/>**TabAddIns**<br/>**TabPrintPreview**<br/>**TabMerge**<br/>**TabGrayscale**<br/>**TabBlackAndWhite**<br/>**TabBroadcastPresentation**<br/>**TabSlideMaster**<br/>**TabHandoutMaster**<br/>**TabNotesMaster**<br/>**TabBackgroundRemoval**<br/>**TabSlideMasterHome**<br/> |
    
-
-## Additional resources
 <a name="bk_addresources"> </a>
 
--  [Add-in commands for Excel, Word and PowerPoint (Preview)](http://msdn.microsoft.com/library/5c898275-e47f-4e97-97df-098b769620d6%28Office.15%29.aspx)
-    
-  
--  [Define add-in commands in your Outlook add-in manifest](http://msdn.microsoft.com/library/4d57d987-cb49-484d-8a54-a55c2850c2aa%28Office.15%29.aspx)
+## See also
+
+-  [Add-in commands for Excel, Word, and PowerPoint](https://docs.microsoft.com/en-us/office/dev/add-ins/design/add-in-commands) 
```

Comment:
For future reference, don't include "en-us" in the URL since that breaks loc navigation

Reply:
Thanks.